### ISSUE TITLE
CVEI convenience functions

### ISSUE THREAD

**Issue by mottosso** [pyblish/pyblish-base#288]
### Goal

As [discussed on the forums](http://forums.pyblish.com/t/pyblish-collect-then-validate/261/4), these functions are transitioning from being practically useless to something more interesting.

**Before**

``` python
# Each function implicitly ran those before it.
# Resulting in "double transformation" when used together.
from pyblish import util

context = util.collect()
context = util.validate()
context = util.extract()
context = util.integrate()
```

**After**

``` python
# They can now be used together
from pyblish import util, api

context = api.Context()

util.collect(context)
util.validate(context)
# Optionally, manually check for errors here..
util.extract(context)
util.integrate(context)
```

**Comment by BigRoy** [comment]
Sweet. Tested the #289 PR and seems smooth.

Here's one that I tested to only publish instances of the `camera` family:

``` python
# They can now be used together
from pyblish import util, api

context = api.Context()

util.collect(context)
context[:] = [instance for instance in context if instance.data['family'] == "camera"]

util.validate(context)
util.extract(context)
util.integrate(context)
```

Or to remove a specific instance by its name whilst keeping the rest:

``` python
# They can now be used together
from pyblish import util, api

context = api.Context()

util.collect(context)

for instance in context[:]:
    if instance.name == "model_default_INST":
        context.remove(instance)

util.validate(context)
util.extract(context)
util.integrate(context)
```

Both seemed to work as expected.

**Comment by mottosso** [comment]
Thanks Roy!

**Comment by mottosso** [comment]
Implemented, will be included in the next release, 1.4.2
